app/dao/service_callback_dao.py

```diff
@@ -0,0 +1,14 @@
+from notifications_utils.statsd_decorators import statsd
+from app.models import (ServiceCallback)
+
+
+@statsd(namespace="dao")
+def dao_get_callback_include_payload_status(service_id, service_callback_type) -> bool:
+
+    row = ServiceCallback\
+        .query.filter_by(service_id=service_id)\
+        .filter_by(callback_type=service_callback_type) \
+        .filter_by(include_provider_payload=True) \
+        .one()
```

Comment:
I reached out to you about this one before, not sure why you left it as a `.one()` call. This should be a `first`. We do not allow exceptions to control the flow of code.